test/integration/proposal-tokenTrade.e2e.ts

```diff
@@ -1,9 +1,9 @@
 import * as uuid from "uuid";
-import { LATEST_ARC_VERSION, hideCookieAcceptWindow, ITestAddresses, gotoDaoPlugins } from "./utils";
+import { LATEST_ARC_VERSION, hideCookieAcceptWindow, gotoDaoPlugins } from "./utils";
 
 describe("Token Trade Proposals", () => {
   let daoAddress: string;
-  let addresses: ITestAddresses;
+  let addresses: any;
```

Comment:
Why this change?

Reply:
@roienatan because interface is incorrect and it lead typescript checks issue. Proper way is just fix interface I gues...

Reply:
@roienatan trying to fix, but interface is deeply different. Code works properly, but we don't have correct interface for it